replicate_manifest_config.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Maybe add a note to the `README.md` with a description and example use of this script

Reply:
Will do.
We gotta make it easy for whoever takes the `release manager` role.